src/components/swap/PendingModalContent/Logos.tsx

```diff
@@ -0,0 +1,101 @@
+import { Currency } from '@uniswap/sdk-core'
+import { LoaderV3 } from 'components/Icons/LoadingSpinner'
+import CurrencyLogo from 'components/Logo/CurrencyLogo'
+import AnimatedConfirmation from 'components/TransactionConfirmationModal/AnimatedConfirmation'
+import styled, { css, keyframes } from 'styled-components/macro'
+
+import { ReactComponent as PapersIcon } from '../../../assets/svg/papers-text.svg'
```

Comment:
copied from `src/components/swap/PendingModalContent.tsx`

Reply:
can this be an absolute import instead of relative 😅 the number of `../` is giving me an aneurysm